cool-core/src/main/java/com/nus/cool/core/schema/FieldType.java

```diff
@@ -77,8 +77,22 @@ public static boolean IsHashType(FieldType fieldType) {
       case Action:
       case AppKey:
       case Segment:
+        return true;
+      case UserKey:
+      case ActionTime:
+      case Metric:
+        return false;
+      default:
+        throw new IllegalArgumentException("Invalid field" + fieldType.toString());
+    }
+  }
+  public static boolean IsInvariantType(FieldType fieldType) {
+    switch (fieldType) {
       case UserKey:
```

Comment:
why does only support the Invariant type for the UserKey? Segment and metrics also should be supported.

Reply:
I rename the function to IsUserKeyField. This function is implemented for metaChunk. In metaChunk, Only the UserKey field may contain information for invariant data.